exercises/nucleotide-count/src/example.c

```diff
@@ -0,0 +1,44 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include "nucleotide_count.h"
+
+char *count(const char *dna_strand)
+{
+   bool invalidChar = false;
+   uint16_t index;
```

Comment:
-- agnostic changed -
complicates printf %zd versus %u -- but does allow you to avoid stdint.h. 
memory size much larger than needed for tests...